src/olympia/reviewers/utils.py

```diff
@@ -1124,6 +1127,11 @@ def log_sandbox_message(self):
 
 class ReviewUnlisted(ReviewBase):
 
+    def clear_all_needs_human_review_flags(self):
+        """Clear needs_human_review flags, but unlike listed review, only do
+        it on the latest version, not all past versions."""
+        self.clear_specific_needs_human_review_flags(self.version)
```

Comment:
I don't understand why it's necessary to override this to change the behaviour - all of the methods that set the flags are overridden anyway?  (maybe just `raise` if it's just for safety)

Reply:
Good catch. I removed the override, simplifies things further 4792137f946c3f337b1f2bc1734cbfc78a9c04c3